src/SvgUtils.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+import { BlissSVGBuilder } from "bliss-svg-builder";
+import { BciAvId } from "./BlissSymbol";
+import { adaptivePaletteGlobals } from "./GlobalData";
+
+export const DEFAULT_SVG_MARKUP_STRING = "B3";   // Question mark
+
+/**
+ * Convert the given `BciAvId` to a SVG builder code string.  If the `BciAvId`
+ * argument is an array of BCI-AV-IDs and punctuation, concatenate the array
+ * into a string of builder code strings and punctuation marks.
+ * @param {BciAvId} - The BciAvId to convert.
+ * @return {String} - The concatenation of the builder codes and punctuation,
+ *                    e.g., "B106/B12".
+ */
+export function bciAvIdToString (bciAvId: BciAvId) {
+  let finalCode = "";
+  if (typeof bciAvId === "number") {
+    const { blissSvgBuilderCode } = bciToBlissaryId(bciAvId);
+    finalCode = blissSvgBuilderCode;
+  }
+  // `bicAvId` is an array
+  else {
+    bciAvId.forEach((item) => {
+      if (typeof item === "number") {
+        const { blissSvgBuilderCode } = bciToBlissaryId(item);
+        finalCode = `${finalCode}${blissSvgBuilderCode}`;
+      } else {
+        finalCode = `${finalCode}${item}`;
+      }
+    });
+  }
+  return finalCode;
```

Comment:
When the corresponding Bliss ID is not found for a BCI-AV-ID, it might be better to `console.erro()` the error so the developer knows which BCI-AV-ID is problematic.

Reply:
There is a `console.error()` where the exception occurs, inside `getSvgString()` (and also in the new `getSvgElement()`).

This might be related:  I don't think `bciToBlissaryId()` should be exported, since it is only used within the SvgUtils module.  The reason that is was exported it was solely for the purpose of writing tests for it.  I have since looked into testing "private" functions and there is more about that in my response below about the addition of tests for the `Palette` component.

If `bciToBlissaryId()` is private, then the only place where the throw will occur is in the two `getSvg...()` functions, where the `console.error()` logs the problem

Reply:
Got it, and great to know the error is already reported.